draft-ietf-6man-slaac-renum-08.xml

```diff
@@ -105,6 +117,17 @@ the title) for use on http://www.rfc-editor.org/search.html. -->
 </section>
 
 
+<section title="Terminology" anchor="term">
+
+<list style="symbols">
+<t>DHCPv6-PD: DHCPv6 Prefix Delegation <xref target="RFC8415"/>; a mechanism to delegate IPv6 prefixes to clients.</t>
+<t>Flash renumbering: a network renumbering event, when an old prefix, used to address hosts, becomes invalid and is replaced by a new prefix. Before the flash renumbering only the old prefix provides connectivity, and after the flash renumbering only the new one can be used. In other words, there is no period of time when addresses from both prefixes provide connectivity. Examples of flash renumbering include, but are not limited to a change of prefix delegated via DHCPv6-PD, or removal of one prefix from the router configuration and replacing it with another.</t>
```

Comment:
RFC8978 doesn't have the terminology section, so I basically copied the definition from RFC8978 introduction:

- RFC8978 Introduction: "However, there are several scenarios that may lead to the so-called "flash-renumbering" events, where a prefix employed by a network suddenly becomes invalid and replaced by a new prefix."
- this PR: Flash renumbering: a network renumbering event, when an old prefix, used to address hosts, becomes invalid and is replaced by a new prefix. Before the flash renumbering only the old prefix provides connectivity, and after the flash renumbering only the new one can be used. In other words, there is no period of time when addresses from both prefixes provide connectivity.
Do you think we re-defining? I was trying to keep the definition in sync.

Reply:
>I would add a reference to RFC8978, noting that it describes multiple scenarios.

Done, PTAL.

>Minor comment: Not sure I'd necessarily say "the prefix is replaced by e new prefix" (even id that's normally the case): In a dual stack scenarios, if the IPv6 prefix disappears, that's also bad: hosts may try IPv6 rather than IPv4, and fail.

Well, I actually copied that definition from RFC8978:

"However, there are several scenarios that may lead to the so-called "flash-renumbering" events, where a prefix employed by a network suddenly becomes invalid and replaced by a new prefix."

Also, "re-" in renumbering kind of implies replacement, am I wrong? But anyway, I've rephrased as:

"Flash renumbering: a network renumbering event, when an old prefix, used to address hosts, becomes invalid and is replaced by a new prefix (or just removed, without any replacement)."

Better?

P.S. I'll copy those changes into PR#2 in case we want to go down that road.